assets/sass/components/adsense/_googlesitekit-adsense-ad-blocking-recovery.scss

```
@@ -39,6 +39,12 @@
 		}
 	}
 
+	.googlesitekit-ad-blocking-recovery__steps {
+		.googlesitekit-stepper__step-title {
+			letter-spacing: 0.1px;
```

Let's use the `$ls-xxs` variable.
```suggestion
			letter-spacing: $ls-xxs;
```